src/sfizz/Interpolators.hpp

```diff
@@ -68,8 +68,8 @@ class Interpolator<kInterpolatorHermite3, R>
     static inline R process(const R* values, R coeff)
     {
         R y = 0;
-        for (int i = 0; i < 4; ++i) {
-            R h = hermite3<R>(i - 1 - coeff);
+        for (int i = -1; i < 3; ++i) {
```

Comment:
To offset it by -1, why not, at it looks correct overall. On the other hand, it needs a new prerequisite for use.

BM_interpolators was not updated to work in this new model. It's going to read out-of-bounds at the start of input.

Reply:
I added a docstring for the padding and updated the benchmark to heed the excessFileFrames